packages/ember-views/lib/mixins/view_support.js

```diff
@@ -610,6 +613,7 @@ export default Mixin.create({
     this.scheduledRevalidation = false;
```

Comment:
I'm not sure I follow what you mean.  This assignment is happening just before `this._super` is called, do you want to move it afterwards or something else?